src/test/features/claim/routes/check-and-send.ts

```diff
@@ -85,10 +85,10 @@ describe('Claim issue: check and send page', () => {
           .get(ClaimPaths.checkAndSendPage.uri)
```

Comment:
Wouldn't it be good to have '<a href='.....>' in a variable and use it in multiple places.?

Reply:
we have different types of <a href= ....> It will be nice to leave it AS-IS instead of them into variable. approximately we need to create 8 different variables to 

![image](https://user-images.githubusercontent.com/41957682/89567511-b0c98400-d819-11ea-8ff5-b25542902c72.png)

![image](https://user-images.githubusercontent.com/41957682/89567547-c3dc5400-d819-11ea-9371-b886da745144.png)

Reply:
![image](https://user-images.githubusercontent.com/41957682/89568675-8c6ea700-d81b-11ea-8f4e-8d12b1758f33.png)